docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,286 @@
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>proto</label>
+        </td>
+        <td>
+          <span className="badge badge--danger">Required</span>
+          The map XML's protocol version.
+        </td>
+        <td>
+          <span className="badge badge--success">Recommended</span>
+          <label>1.4.2</label>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+##### Map Protocol Values
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Version</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>1.4.2</label>
+        </td>
+        <td>
+          Refer to <a href="#changes-in-142">Changes in 1.4.2</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.4.1</label>
+        </td>
+        <td>
+          <span className="badge badge--secondary">Note</span>
+          No change in features on PGM. Use <label>1.4.0</label> or <label>
+            1.4.2
+          </label> instead.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.4.0</label>
+        </td>
+        <td>
+          Filters, regions, and teams are always referenced by its ID (replaces{" "}
+          <label>name</label>).
+          <br />
+          Disallows <label>&lt;time&gt;</label> inside{" "}
+          <label>&lt;score&gt;</label> or <label>&lt;blitz&gt;</label> &
+          disallows <label>&lt;title&gt;</label> inside{" "}
+          <label>&lt;blitz&gt;</label>.
+          <br />
+          Required objectives to define if they are needed to win the match.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.6</label>
+        </td>
+        <td>
+          Moved all defining elements out of module XML root.
+          <br />
+          Everything scores zero points by default.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.5</label>
+        </td>
+        <td>Filters are aware of who owns TNT.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.4</label>
+        </td>
+        <td>Wools must have a location property.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.3</label>
+        </td>
+        <td>Define how overlapping regions should behave.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.2</label>
+        </td>
+        <td>Added monument modes.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.3.1</label>
+        </td>
+        <td>Off-by-one region bug fixed.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Map Protocol Changelog
+
+### Changes in 1.4.2
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- The objectives filter will now always automatically derive team ownership for the objective from context. To match an objective without any specific team, the <label>any="true"</label> attribute needs to be specified.
```

Comment:
`<label>any="true"</label>` is not being properly highlighted but this is most likely a problem with Docusarus or some other quirk involving the `<span>` tag. This can be fixed later so you can leave it as is

Reply:
Switching it to use \`<label>any="true"</label>` could work, but it will not have the red highlight as it appear in the table.